src/microbe_stage/editor/EditorComponentBottomLeftButtons.tscn

```diff
@@ -170,6 +170,7 @@ size_flags_vertical = 4
 custom_colors/font_color = Color( 1, 1, 1, 1 )
 max_length = 100
 placeholder_text = "SPECIES_NAME_DOT_DOT_DOT"
+caret_blink = true
```

Comment:
Is this a changed option or is this like the default value? Godot editor usually when you reopen it, it will remove default values.

Reply:
I think this is a changed option, personally I like if the caret blinks so this is a welcome change from me.

Reply:
I suppose a blinking caret is the more common way it usually works.

Reply:
This is a value I changed. I think it looks far better. If people disagree that's fine, I can change it.